frontend/src/app/pages/exercises/exercise/shared/emergency-operations-center/emergency-operations-center-full/emergency-operations-center-full.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="modal-body" style="min-height: 60vh">
+    <app-eoc-alarm-group-triggers />
+    <br />
+    <div class="card">
+        <h5 class="card-header">Einsatztagebuch</h5>
+        <div class="card-body">
+            <app-eoc-log-interface />
+        </div>
+    </div>
```

Comment:
I'd prefer if the subcomponents would use the same nesting depth, i.e., both contain only the card body (like `app-eoc-log-interface`) or contain the full card (like `app-eoc-alarm-group-triggers`)